.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  semi: true,
+  plugins: [require.resolve("@trivago/prettier-plugin-sort-imports")],
```

Comment:
Had to change .prettierrc to .prettierrc.js because sort import plugin doesn't work unless if I use `require`. Related: https://github.com/trivago/prettier-plugin-sort-imports/issues/131#issuecomment-1163488213